README.rst

```diff
@@ -41,7 +41,7 @@ Installation
         "post_office",
     )
 
-* Run ``syncdb`` for Django <= 1.8::
+* Run ``syncdb`` for Django == 1.8::
```

Comment:
Actually, I think `python manage.py migrate` exists and `syncdb` is deprecated in 1.7+,  so this can be removed...